examples/03_model_scoring.py

```diff
@@ -137,18 +137,24 @@
 # can then be pose process using some of the utility functions Earth-2 MIP provides.
 
 # %%
+import numpy as np
 from earth2mip.inference_medium_range import save_scores, time_average_metrics
 
-time = datetime.datetime(2017, 1, 1, 0)
-initial_times = [time + datetime.timedelta(days=30 * i) for i in range(12)]
+# Use 52 initializations.
```

Comment:
For the example, please reduce this to at most 12. Designed to just be a sampler, not a full blown analysis

Reply:
Updated, and added more docs